cli/search/search.go

```diff
@@ -115,6 +115,62 @@ func GetVersionsFromGitRemote(repo string) ([]version.Version, error) {
 	return versions, nil
 }
 
+// GetCommitsFromGitLocal returns commits list from specified local git repo.
+func GetCommitFromGitLocal(repo string, input string) (string, error) {
+	if _, err := exec.LookPath("git"); err != nil {
+		return "", fmt.Errorf("unable to get commits: `git` command is missing")
+	}
+
+	if util.IsPullRequest(input) {
+		commandStr := "pull/" + util.GetPullRequestID(input) +
+			"/head:" + input
+		cmd := exec.Command("git", "fetch", "origin", commandStr)
+		cmd.Dir = repo
+		err := cmd.Run()
+		if err != nil {
+			return "", err
+		}
+	}
+
+	cmd := exec.Command("git", "show", input)
```

Comment:
Can we use `git show --quiet` here to output without diff?

Reply:
Thanks for a new flag